lib/puppet/vendor/plist/README

```diff
@@ -0,0 +1,36 @@
+= All-purpose Property List manipulation library
+
+Plist is a library to manipulate Property List files, also known as plists.  It can parse plist files into native Ruby data structures as well as generating new plist files from your Ruby objects.
+
+== Usage
+
+See USAGE[link:files/docs/USAGE.html].
+
+== Links
+
+[<b>Project Page</b>]          http://plist.rubyforge.org
+[<b>Subversion repository</b>] svn://rubyforge.org//var/svn/plist
+[<b>RDoc (on RubyForge)</b>]   http://plist.rubyforge.org
+
+== Credits
+
+plist is maintained by Ben Bleything <mailto:ben@bleything.net> and Patrick May <mailto:patrick@hexane.org>.  Patrick wrote most of the code; Ben is a recent addition to the project, having merged in his plist generation library.
+
+Other folks who have helped along the way:
+
+[<b>Martin Dittus</b>] who pointed out that +Time+ wasn't enough for plist <tt>Dates</tt>, especially those in <tt>~/Library/Cookies/Cookies.plist</tt>
+[<b>Chuck Remes</b>] who pushed Patrick towards implementing <tt>#to_plist</tt>
+[<b>Mat Schaffer</b>] who supplied code and test cases for <tt><data></tt> elements
+[<b>Michael Granger</b>] for encouragement and help
+
+== License and Copyright
+
+plist is released under the MIT License.
```

Comment:
We already have for the last 8 years, so that's a fun time if not. See https://github.com/puppetlabs/facter/blob/46d9bedc8b3f894fc8fab64955775daf94211814/lib/facter/util/plist.rb#L6.